README.md

```diff
@@ -217,3 +217,7 @@ In case you are using ProGuard add the following snippet to your `proguard-rules
 ```
 
 This will prevent issues like [#131](https://github.com/PhilipsHue/flutter_reactive_ble/issues/131) .
+
+#### Unofficial example apps
+
+Example implementation UART over BLE:[link](https://github.com/wolfc01/flutter_reactive_ble_uart_example)
```

Comment:
At least put space after the colon.

Also, consider replacing "link" with something more meaningful, e.g.:

```markdown
[wolfc01/flutter_reactive_ble_uart_example](https://github.com/wolfc01/flutter_reactive_ble_uart_example)
```

Reply:
@werediver  sorry for the confusion actually I should have rebased the null-safety branch.

Reply:
It would be better, indeed. As long as rebasing is not too painful (and if it is, we're anyway in trouble).